src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
```

Comment:
Think about which variables should be local in the method and which should be class-level. In this case, consider implementing `calculateArea` as an overridden method from the `Figure` class, since it's a behavior that all figures should have.